### PR TITLE
added resources button in navbar

### PR DESCRIPTION
**Pull request by shivang1919**
added resources button in navbar which leads to hog resources library.

image

### CODE REVIEW

File: client/src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -106,6 +106,18 @@ function Navbar() {
               Event
             </li>
           </Link>
+          <Link to={{pathname: "https://hgresources.netlify.app/"}} target="_blank" className="navLink">
```
Comment (SrijanSriv) [inline client/src/components/Navbar/Navbar.js]: dont redirect from the tag. you can remove the _blank

File: client/src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -106,6 +106,18 @@ function Navbar() {
...
+              className={active === "resources" ? "nav-item active " : "nav-item "}
+              onClick={() => {
+                setActive("resources");
+                toggleNav(!navOpen);
+              }}
+            >
+              Resources
+            </li>
+          </Link>
+          {/* <a href="https://hgresources.netlify.app/">Resources</a> */}
```
Comment (SrijanSriv) [inline client/src/components/Navbar/Navbar.js]: no need to to keep any comments :)

**Review by SrijanSriv — CHANGES_REQUESTED** [review]
restore `client/package-lock.json`. we dont want different systems to update unwanted dependencies

**Comment by shivang1919** [comment]
added desired changes.

**Review by SrijanSriv — APPROVED** [review]

**Comment by SrijanSriv** [comment]
LGTM thanks a lot @shivang1919 😸

**Pull request merged**